packages/theme-core-plugin--sass/sass.js

```diff
@@ -17,12 +17,56 @@ const join = require('path').join;
 const del = require('del');
 const core = require('theme-core');
 const defaultConfig = require('./config.default');
-// const debug = require('gulp-debug');
+const path = require('path');
+const stylelintConfig = require('./.stylelintrc.js'); // stylelint config rules
+
 
 module.exports = (userConfig) => {
   const config = core.utils.merge({}, defaultConfig, userConfig);
   const tasks = {};
 
+  // PostCSS config. @TODO: consider moving all this over to defaultConfig
+  const processors = [
+    autoprefixer({
+      browsers: config.autoPrefixerBrowsers,
+    })
+  ];
+
+
+  function validateCss(errorShouldExit) {
+    return gulp.src(config.src)
+        .pipe(cached('validate:css'))
+        .pipe(plumber({
+          errorHandler(error) {
+            notify.onError({
+              title: 'CSS Linting <%= error.name %> - Line <%= error.line %>',
+              message: '<%= error.message %>',
+            })(error);
+            if (errorShouldExit) process.exit(1);
+            this.emit('end');
+          },
+        }))
+        .pipe(stylelint({
+          config: stylelintConfig,
```

Comment:
Passing in config like this would prevent a user from having there own config defined via `.stylelintrc` which would be committed in there repo.